docs/large-bucket/DESIGN.md

```diff
@@ -184,7 +201,7 @@ type formatXLV2 struct {
 }
 ```
 
-## Limits
-
-- Minimum of 4 disks are needed for erasure coded configuration.
-- Maximum of 32 distinct nodes are supported in distributed configuration.
+## <a name="limits"></a>5. Limits
+Large-bucket support has the following limitations:
```

Comment:
These are not limitations but just `limits` - the word limitations signify entirely different context than `limits`.